flux_led/utils.py

```diff
@@ -132,6 +133,20 @@ def percentToByte(percent):
         return int((percent * 255) / 100)
 
 
+def rgbww_to_rgbcw(
+    rgbww_data: Tuple[int, int, int, int, int]
+) -> Tuple[int, int, int, int, int]:
+    r, g, b, w, c = rgbww_data
+    return r, g, b, c, w
+
```

Comment:
Can we names these "rgbwc_to_rgbcw" and "rgbcw_to_rgbwc" ?

Reply:
8b7b890eb31dae36a4f32081c68d225494601e6c